addons/website/models/website.py

```diff
@@ -441,6 +441,15 @@ def set_images(images):
 
         return url
 
+    @api.model
+    def _neutralize(self):
+        super()._neutralize()
+        websites_domains = self.search_read([('domain', '!=', False)], ['domain',])
+        self.browse([rec['id'] for rec in websites_domains]).write({'domain': False})
+        for ws in websites_domains:
+            logger.info('Domain `%s` neutralized', ws['domain'])
```

Comment:
Wouldn't this be more idiomatic and easier to read?
```suggestion
        websites = self.search([('domain', '!=', False)])
        domains = websites.mapped('domain')
        websites.write({'domain': False})
        for domain in domains:
            logger.info('Domain %r neutralized', domain)
```